packages/zosjobs/src/ModifyJobs.ts

```diff
@@ -0,0 +1,118 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { AbstractSession, ImperativeExpect, Logger, Headers, ImperativeError } from "@zowe/imperative";
+import { JobsConstants } from "./JobsConstants";
+import { ZosmfRestClient } from "@zowe/core-for-zowe-sdk";
+import { IJob } from "./doc/response/IJob";
+import { IModifyJobParms } from "./doc/input/IModifyJobParms";
+import { IModifyJob } from "./doc/input/IModifyJob";
+import { IJobFeedback } from "./doc/response/IJobFeedback";
+
+
+/**
+ * Class to handle modify of jobclass and holdStatus information
+ * @export
+ * @class ModifyJobs
+ */
+export class ModifyJobs {
+
+    /**
+     * Modify a job
+     * @static
+     * @param {AbstractSession} session - z/OSMF connection info
+     * @param {string} jobname - job name to be translated into parms object
+     * @param {string} jobid - job id to be translated into parms object
+     * @param {string} jobclass - job class to be translated into parms object
```

Comment:
I would agree that we should probably pass an object to specify options for `modifyJob`, especially since an options object is constructed anyway before `modifyJobCommon` is called. By standardizing to one options object, this would prevent potential mishaps when calling this function internally - such as a missing `undefined` in the parameter list, or if the function was called with `hold/release` passed in the wrong order.